Client/Frontend/Login/LoginInfoTableViewCell.swift

```diff
@@ -0,0 +1,205 @@
+        configureLayout()
+    }
+
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    override func prepareForReuse() {
+        super.prepareForReuse()
+        delegate = nil
+        
+        descriptionTextField.do {
+            $0.isSecureTextEntry = false
+            $0.keyboardType = .default
+            $0.returnKeyType = .default
+            $0.isUserInteractionEnabled = false
+        }
+    }
+    
+    override func canPerformAction(_ action: Selector, withSender sender: Any?) -> Bool {
+        return delegate?.canPerform(action: action, for: self) ?? false
+    }
+    
+    // MARK: Internal
+    
+    private func applyTheme() {
+        backgroundColor = .secondaryBraveBackground
+        selectionStyle = .none
+        separatorInset = .zero
+        
+        descriptionTextField.do {
+            $0.textColor = .braveLabel
+            $0.font = .preferredFont(forTextStyle: .callout)
+            $0.adjustsFontForContentSizeCategory = true
+            $0.isUserInteractionEnabled = false
+            $0.autocapitalizationType = .none
+            $0.autocorrectionType = .no
+            $0.accessibilityElementsHidden = true
+            $0.adjustsFontSizeToFitWidth = false
+            $0.delegate = self
+            $0.isAccessibilityElement = true
+        }
+        
+        highlightedLabel.do {
+            $0.font = .preferredFont(forTextStyle: .footnote)
+            $0.adjustsFontForContentSizeCategory = true
+            $0.textColor = .secondaryBraveLabel
+            $0.numberOfLines = 1
+        }
+    }
+
+    private  func configureLayout() {
+        labelContainer.addSubview(highlightedLabel)
+        labelContainer.addSubview(descriptionTextField)
+        contentView.addSubview(labelContainer)
+        
+        labelContainer.snp.remakeConstraints { make in
+            make.centerY.equalTo(contentView)
+            make.trailing.equalTo(contentView).offset(-UX.horizontalOffset)
+            make.leading.equalTo(contentView).offset(UX.horizontalOffset)
+        }
+
+        highlightedLabel.snp.remakeConstraints { make in
+            make.leading.top.equalTo(labelContainer)
+            make.bottom.equalTo(descriptionTextField.snp.top)
+            make.width.equalTo(labelContainer)
+        }
+
+        descriptionTextField.snp.remakeConstraints { make in
+            make.leading.bottom.equalTo(labelContainer)
+            make.top.equalTo(highlightedLabel.snp.bottom)
+            make.width.equalTo(labelContainer)
+        }
+
+        setNeedsUpdateConstraints()
+    }
+}
+
+// MARK: MenuHelperInterface
+
+extension LoginInfoTableViewCell: MenuHelperInterface {
+
+    func menuHelperReveal() {
+        displayDescriptionAsPassword = false
+    }
+
+    func menuHelperSecure() {
+        displayDescriptionAsPassword = true
+    }
+
+    func menuHelperCopy() {
+        UIPasteboard.general.string = descriptionTextField.text
```

Comment:
nit: currently the copied password stays on the clipboard indefinitely, `expirationDate` can be used to expire clipboard contents. what do you think about setting it to something like 5-10 mins?

Reply:
Expiry text 300 sec is added

at https://github.com/brave/brave-ios/pull/4124/commits/3d37662de9abada3cdaa1abc49af6bdc2d349044